env0/resource_environment.go

```diff
@@ -380,6 +385,13 @@ func setEnvironmentSchema(ctx context.Context, d *schema.ResourceData, environme
 		return fmt.Errorf("schema resource data serialization failed: %v", err)
 	}
 
+	//lint:ignore SA1019 reason: https://github.com/hashicorp/terraform-plugin-sdk/issues/817
+	if _, exists := d.GetOkExists("vcs_pr_comments_enabled"); !exists {
```

Comment:
if not configured in the schema ignore whatever is returned in the response.
This is not perfect, but will work well for customers.